scripts/benchmark_utils.py

```
@@ -406,6 +461,38 @@ def _run_circumsphere_benchmarks(self, cargo_profile: str | None = None) -> tupl
+
+            run_cargo_command(
+                cargo_args,
+                cwd=self.project_root,
+                timeout=900,
+                capture_output=True,
+            )
+
+            print("✅ ci_performance_suite benchmarks completed successfully")
+            return True
+
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)